### ISSUE TITLE
graphql-import does not work with Apollo Federation

### ISSUE THREAD

**Issue by villesau** [ardatan/graphql-import#352]
HI, when building federated schema using basic builiding blocks, following error is thrown:

```
error: Uncaught exception: (message=Directive key: Couldn't find type key in any of the schemas., stack=Error: Directive key: Couldn't find type key in any of the schemas.
    at collectDirective (/Users/user/code/node_modules/graphql-import/src/definition.ts:172:15)
    at Array.forEach (<anonymous>)
    at collectNewTypeDefinitions (/Users/user/code/node_modules/graphql-import/src/definition.ts:86:30)
    at Object.completeDefinitionPool (/Users/user/code/node_modules/graphql-import/src/definition.ts:49:39)
    at Object.importSchema (/Users/user/code/node_modules/graphql-import/src/index.ts:127:18)
```

The type that causes the issue looks like this:

```
type SomeType @key(fields: "id") {
  id: ID!
}
```

key directive is something that glues different pieces of extended types together when federating schema. See the documentation for further details: https://www.apollographql.com/docs/apollo-server/federation/core-concepts/

Could the validation be loosened on import-schema library so that `key` would not cause exception?

**Comment by ardatan** [comment]
Hi @villesau !
In 1.0.0 beta release, we introduced a lot of changes including apollo federation support;
Could you install `graphql-import@beta` to try new changes? Don't forget to modify your code regarding to the migration notes in `README`. 
https://github.com/ardatan/graphql-import#updating-from-07x

**Comment by villesau** [comment]
Thanks @ardatan ! Unfortunately we postponed our federation plans so I'm not able to test it any time soon. I hope someone else who has faced this issue can report back if the changes works for them!

**Comment by ardatan** [comment]
Available in 1.0.0!
